src/game_constants.h

```diff
@@ -82,7 +82,13 @@
 /** Weight per level of LIFT/JACK tool quality */
 #define TOOL_LIFT_FACTOR 500000 // 500kg/level
 
+/** Cap JACK requirements to support arbritrarily large vehicles */
+#define JACK_LIMIT 6000000 // 6000kg (6 metric tonnes)
```

Comment:
It's consistent with the rest of the definitions here, but shouldn't they become `static const`s at some point?